externals/kyuubi-spark-sql-engine/src/test/scala/org/apache/kyuubi/engine/spark/WithSparkSQLEngine.scala

```diff
@@ -41,10 +42,11 @@ trait WithSparkSQLEngine extends KyuubiFunSuite {
       s"jdbc:derby:;databaseName=$metastorePath;create=true")
     System.setProperty("spark.sql.warehouse.dir", warehousePath.toString)
     System.setProperty("spark.sql.hive.metastore.sharedPrefixes", "org.apache.hive.jdbc")
+    conf.foreach{case (k, v) => System.setProperty(k, v)}
+    SparkSQLEngine.kyuubiConf.loadFromSystemProperties()
```

Comment:
Do we still need this? seems `KyuubiConf(loadSysDefault: Boolean = true)` is able to load sys props?

Reply:
We need this due to `SparkSQLEngine.kyuubiConf` is a global static field.

Reply:
```scala
conf.foreach{case (k, v) =>
  System.setProperty(k, v)
  SparkSQLEngine.kyuubiConf.set(k, v)
}
```

Is it much more clear?

Reply:
ok